packages/manager/modules/pci/src/components/project/regions-list/regions-list.html

```diff
@@ -26,6 +12,50 @@
                         }">
                     </span>
                 </p>
+
+                <div class="col-md-6 col-xl-4 my-3" data-ng-repeat="(macro, regions) in $ctrl.regionsByContinents[continent]">
+                    <oui-select-picker
+                        data-name="{{continent}}_regions"
+                        data-model="$ctrl.macroRegion"
```

Comment:
```suggestion
                        data-model=":: $ctrl.macroRegion"
```
and so on

Reply:
I'm not sure we want `model` to be one-time binded

Reply:
Indeed